### DISCUSSION TITLE
Changing the tag name for the manual for future releases

### DISCUSSION THREAD

**Discussion by ReinerBforartists** [Bforartists/Bforartists#3373] [General]
This is a note for doing a manual release at github in the future.

We have used the same tag name than the branch in the past. It was convenient. But Github then moaned about duplicate tag name. I have now chosen a more unique tag name for this release. manual_xxxxx . Where the xxxx stands for the release branch name now. This avoids the moaning from github.

![tagname](https://user-images.githubusercontent.com/13919702/212464384-454e8253-d8a9-40ef-9288-e607f62d8182.jpg)

**Comment by Draise14** [comment]
That's a good idea. It does moan a lot.

**Reply by ReinerBforartists** [reply]
It did not moan when i did it the first time :D

Now it does, so time for a change :)
